CRM/Core/Payment/Form.php

```diff
@@ -360,6 +360,15 @@ public static function validateCreditCard($values, &$errors, $processorID = NULL
    * @param bool $reverse
    */
   public static function mapParams($id, $src, &$dst, $reverse = FALSE) {
+    // Set text version of state & country if present.
```

Comment:
@seamuslee001 can you check if it still works if I remove these lines again - my theory is they can go again now...